corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html

```diff
@@ -11,7 +11,7 @@
     </a>
 
     <a id="view_form_{{ module.unique_id }}_{{ form.unique_id }}_sidebar"
-       {% if form.no_vellum %}
+       {% if form.form_type == 'shadow_form' %}
```

Comment:
Is this a new feature: shadow forms do not get vellum?

Reply:
It's a few years old. Shadow forms are a variant of advanced forms. They're tied to a "source" form, and they use that form's XML and case actions but can also add their own case actions. They were using the `no_vellum` attribute, so I'm just switching to instead check the form type directly, it shouldn't change their behavior.

Reply:
Maybe the `no_vellum` field should be kept for just this purpose? Form types that cannot be edited in vellum can set a class attribute `no_vellum = True` (might be nice to reverse that logic to `can_edit_in_vellum = False` and have the default be `True`).

Reply:
Is this a couch field? If so, it should be private (with a leading underscore) so it is not saved in Couch.

Reply:
I'm not following this. `form_type` is a couch field, yes, it's a property of `FormBase`. What do you mean about it not being saved in couch? I'm not keen on keeping `no_vellum` as its own attribute if it's synonymous with `form_type == 'shadow_form'`

Reply:
I'd be fine with making a `FormBase.can_edit_in_vellum` property for readability, I'd just like to get rid of `no_vellum` as a data field.

Reply:
A `can_edit_in_vellum` property sounds good to me.

Reply:
Cool, updated in 45855d65819

Reply:
Thanks!